public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -614,6 +639,12 @@ public SurveyQuestionResult apply(AchillesResult o) {
         }
 
         List<SurveyModule> surveyModules = surveyModuleDao.findSurveyModuleQuestionCounts(surveyKeyword);
+
+        if (domains == null || surveyModules == null) {
```

Comment:
This will never be true, since `surveyModuleDao.findSurveyModuleQuestionCounts(surveyKeyword)` will never return null(only empty list).

Reply:
This is for the case when the domain info and survey module tables itself are not present.

Reply:
surveyModules will never be null and should be removed from the form the if or you should add `surveyModules.isEmpty()` instead.